nemo/collections/asr/parts/utils/vad_utils.py

```diff
@@ -92,15 +94,16 @@ def prepare_manifest(config):
     return manifest_vad_input
 
 
-def write_vad_infer_manifest(file, args_func):
+def write_vad_infer_manifest(file: dict, args_func: dict) -> list:
     """
     Used by prepare_manifest.
     Given a list of files, write them to manifest for dataloader with restrictions.
```

Comment:
what do restrictions mean here?

Reply:
oh good point. it means with at most split_duration duration. Let me clarify it.